app/population/middleware/set-filter.population.js

```diff
@@ -0,0 +1,16 @@
+function setFilterPopulation(items = [], bodyKey) {
+  return function buildFilter(req, res, next) {
+    const { locations } = req
+    const locationIds = locations?.join()
+
+    const filter = {
+      'filter[location_id]': locationIds,
```

Comment:
Similar to the previous comment, I think it would be good to isolate the API specific filters in the service rather than within the middleware. And [pass `locationIds`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/ef40ccf4dbe16dc4919e204ea43d9c8c250e8a3f/common/services/move.js#L112) in to the service and call the filter from [within that method](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/ef40ccf4dbe16dc4919e204ea43d9c8c250e8a3f/common/services/move.js#L133)